haskell/hsc2hs.bzl

```diff
@@ -49,6 +49,14 @@ def _process_hsc_file(ctx, hsc_file):
   hdrs, include_args = cc_headers(ctx)
   args.add(include_args)
 
+  # One of the standard definitions that should be available, we'll add them
```

Comment:
This comment should be updated to point to `platformDefines` in `Cabal/Distribution/Simple/PreProcess.hs`.